README.md

```diff
@@ -1,2 +1,55 @@
-[![Join the chat at https://gitter.im/open-telemetry/opentelemetry-ruby](https://badges.gitter.im/open-telemetry/opentelemetry-ruby.svg)](https://gitter.im/open-telemetry/opentelemetry-ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Build Status](https://travis-ci.org/open-telemetry/opentelemetry-ruby.svg?branch=master)](https://travis-ci.org/open-telemetry/opentelemetry-ruby)
+# OpenTelemetry Ruby
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![Travis CI][ci-image]][ci-url]
+[![Apache License][license-image]][license-image]
+
+The Ruby [OpenTelemetry](https://opentelemetry.io/) client.
+
+## Contributing
+
+We'd love your help! Use tags [good first issue][issues-good-first-issue] and
+[help wanted][issues-help-wanted] to get started with the project.
+
+The Ruby special interest group (SIG) meets regularly. See the OpenTelemetry
+[community page][ruby-sig] repo for information on this and other language SIGs.
+
+
+## Release Schedule
+
+OpenTelemetry Ruby is under active development. Our goal is to release an
+_alpha_ version of the library at the end of September 2019. This release isn't
+guaranteed to conform to a specific version of the specification, and future
+releases will not attempt to maintain backwards compatibility with the alpha
+release.
+
+| Component                   | Version | Target Date       |
+| --------------------------- | ------- | ----------------- |
+| Tracing API                 | Alpha   | Unknown           |
+| Tracing SDK                 | Alpha   | Unknown           |
+| Metrics API                 | Alpha   | Unknown           |
+| Metrics SDK                 | Alpha   | Unknown           |
+| Jaeger Trace Exporter       | Alpha   | Unknown           |
```

Comment:
Leave this as Unknown for now. I started scoping the work today, but I'm not sure exactly what's expected (UDP export to the agent only? Thrift? Protobuf? grpc? HTTP?). AFAICT we don't have an existing Jaeger client for Ruby to piggyback on.

Reply:
Perhaps the [jaeger-client-ruby](https://github.com/salemove/jaeger-client-ruby) lib linked in [this issue](https://github.com/jaegertracing/jaeger/issues/268) has something we can use. From my quick glance this looks more like an OpenTracing Tracer than a Jaeger client library.

Reply:
Thanks for the link! We can definitely use that, either as a starting point or we can wrap it. I’ll dig into it some more.